crates/iceberg/src/scan.rs

```diff
@@ -445,4 +494,22 @@ mod tests {
             format!("{}/3.parquet", &fixture.table_location)
         );
     }
+
+    #[tokio::test]
+    async fn test_open_parquet_no_deletions() {
```

Comment:
While this is currently fine for unit tests, it would be better to add integrations for this.

Reply:
Thanks - I'll create an integration test in the follow-up PRs that extend the functionality